builds/e2e/connectivity.yaml

```diff
@@ -12,10 +12,10 @@ schedules:
     - main
   always: true
 - cron: "0 12 * * 1,2,3,4,5,6"
-  displayName: Daily run release/1.3
+  displayName: Daily run release/1.4
   branches:
     include:
-    - release/1.3
```

Comment:
Please don't forget to remove this from release/1.3 branch as well or it will be a run collision.

Reply:
Correct. Ideally we'd just delete the branch and all schedule triggers for that branch would stop. But following existing patterns, we'll remove the include in release/1.3 branch.

Reply:
See #6612